ui/src/app/base/sagas/websockets.ts

```diff
@@ -0,0 +1,627 @@
+import type {
```

Comment:
Not sure why this is showing up as a new file instead of a rename. It might help to manually diff this against the old file.